code/modules/mob/living/carbon/brain/robotic_brain.dm

```diff
@@ -26,6 +26,9 @@
 	return ..()
 
 /obj/item/mmi/robotic_brain/attack_self(mob/user)
+	if(isgolem(user))
+		to_chat(user, "<span class='warning'>[src] seems to require an organic or robotic user. You can't find a way to activate it.")
```

Comment:
Suggestion: it's stated that golems can't use guns because of their fat fingers/hands. You think it might work that they can't activate the switch because their fingers are too thick to reach it?

Reply:
sure why not